src/pages/Layout/Layout.jsx

```diff
@@ -0,0 +1,15 @@
+import "./layout.scss";
+
+const Layout = ({ renderHeader = false, header, children }) => {
+  return (
+    <section className="layout flex flex-col">
+      {renderHeader ? (
+        <div className="layout__header b-margin-md"> {header} </div>
+      ) : null}
+      <div className="layout__body">{children}</div>
+      {/* <button className="btn">Logout</button> */}
```

Comment:
you can remove commented code if not required

Reply:
I will use it in next PR :)